docs/en/OLED_Display.md

```diff
@@ -0,0 +1,199 @@
+# Oled Display
+Extension to use for your build with OLED Display
+
+If you need help soldering, there's a small note on the end to help.
+
+# Preparation
+
+### Libraries
+First of all you need to download few Libraries that will make possible for your OLED to work. You can download them from Adafruit CircuitPython Libraries page. 
+#### Make sure you download right version of Libraries. It should be the same one as yours CircuitPyhon.
+* [CircuitPython Libraries Bundle](https://circuitpython.org/libraries)
+
+Now you need to create `lib` folder where you will place your needed files.
+Then find following folder and file and drop them in your freshly baked folder.
+* adafruit_display_text
+* `adafruit_displayio_ssd1306.mpy`
```

Comment:
```suggestion
Then find following folder and file and drop them in your freshly baked folder:
* `adafruit_display_text`
* `adafruit_displayio_ssd1306.mpy`
```
Use consistent backticks for inline code highlighting. This is not the only occurance, check the rest of this document for any code pieces, file names, module names, etc., that should be highlighted with backticks.